.github/workflows/ocaml.yml

```diff
@@ -0,0 +1,54 @@
+name: OCaml
+
+on:
+  pull_request:
+  push:
+    branches: ["master"]
```

Comment:
Hello @Yag000,
Thank you very much for the proposal 😋 !
Please could you just make this one run everywhere ?
Have a good day!

Reply:
What about [e8b459b](https://github.com/Makyou-Nasou/Chess/pull/1/commits/e8b459b73be272ea49a466e14c6cdb2a9e18f8d2) ? It should run on every push event on this repository.

Reply:
Thank you very much! That’s very nice of you.